File: Ex1.js

function isPrime(number) {
    if (number < 2) {
        return false
    }
    for (let i = 2; i < number; i++) {
        if (number % i == 0) {
            return false
        }
    }
    return true
}


console.log(isPrime(13));


let arr = [1, 2, 11, 5, 8, 4, 3, 9, 15]
max = 0

for (i = 0; i < arr.length; i++) {
    if (isPrime(arr[i]) && max < arr[i]) {
        max = arr[i]
    } 
}
console.log(max);

File: Ex1.py
import math

def is_prime(n):
    for i in range(2, int(math.sqrt(n)) + 1):
        if n % i == 0:
            return False
    return True

a = [1,2,3,4,7,6,3,11]


if __name__ == "__main__":
    max = 0
    for i in a:
        if is_prime(i):
            if max < i:
                max = i

    print(max)
